src/Views/ProgressView.vala

```diff
@@ -16,6 +16,8 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+using Distinst;
```

Comment:
We tend to avoid `using` to make conflicting namespaces less ambiguous. Not sure if that'd be a blocker, but regardless I want to get someone more familiar with the Installer code to give this a look.

Reply:
I removed it in https://github.com/elementary/installer/pull/222/commits/9826b6c3b43345bc76961faf336be9cae36acc4b